src/LibraryManager/Manifest.cs

```diff
@@ -228,15 +228,18 @@ private int DeleteLibraryFiles(ILibraryInstallationState state, Action<string> d
         {
             int filesDeleted = 0;
 
-            foreach (string file in state.Files)
+            if (state.Files != null)
```

Comment:
This is not a part of this PR, right? Also we need a bit more complicated fix here. Rather than doing nothing, we need to get full file list from the catalog. "null" here means "all files".

Reply:
Good catch... I had it locally to avoid crashes while I was testing... :)